### PR TITLE
Added images and readme file (index.md)

### PR DESCRIPTION
**Pull request by dylansutterlin**
Pull request to merge with my project template, images and description. Let me know if anything in the markdown formatting has gone wrong! Thank you

### CODE REVIEW

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+summary: "Painful experience involves a distributed pattern of brain activity. With hypnosis, it's possible to increase or decrease pain. This project aims to decode fMRI pain-evoked brain activity and identify pattern of activity that are associated with specific hypnotic conditions"
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "brain-activity.webp"
+---
+<!-- This is an html comment and this won't appear in the rendered page. You are now editing the "content" area, the core of your description. Everything that you can do in markdown is allowed below. We added a couple of comments to guide your through documenting your progress. -->
+
+
+### Background
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:31]: Please remove the empty sections from the template

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+The tools used to accomplish this task are:
+
+* Python scripts
+* Python modules (pandas, numpy, sklearn, nilearn, nibabel,  matplotlib, seaborn)
+* Bash to run the scripts on elm server
+* Git and Github to keep track of the project's evolution
+
+### Data
+
+The dataset that will be used comes from Desmartaux et al., 2021 and is access restricted. It includes 24 participants (13 women and 11 males) and mean age is 26.9. Subjects participated to a fMRI scanning session where they received hypnosis to either increase or decrease pain. After hypnotic suggestions to modulate pain,a serie of either 6 or 9 painful stimuli were administered. In total, each participant received 72 electrical shocks. Across all participants, a total of 1728 trials/shocks were done.
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:85]: has restricted access

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+* - [x] Python scripts (main script that run the analyses)
+* - [x] Metrics and statistics of the performances of the model
+* - [x] Graphics (e.g. ROC curve, confusion matrix)
+* - [x] fMRI maps of the voxels that best predicted suggestions plotted as 3D interactive plots
+* - [x] A markdown README.md describing globaly the project and putting it in a scientific context
+* - [x] Github repository provinding a fairly open science/reproductible aspect to the project
+
+#### Final script's structure
+* data :vPermission pending to give access to date (...)
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:129]: typo

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+* docs : A readme file descripbing the pipeline
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:130]: typo

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+* images : The images used in this README.me
+* results : Saved metrics' dataframe, brain maps (.nii files) showing the voxels that were the most predictive of the target
+* scripts : The main scripts with a function file that are called in main.py
+* src : All the general function that can we reused wihout context
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:134]: typo

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+### Statistical model
+
+* The SVC model was performed with a linear kernel
+* A nifti masker was used to project the 3 dimmensional contrast maps to 1 dimension with the argument `(mask_strategy = 'whole-brain-template', standardize = True)`
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:139]: typo dimensional

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+
+* A Matrix confusion is presented to illustrate the model'performance
+*  * Class 1 = Hyperalgesia
+   * Class 2 = Neutral suggestion in Hyperalgesia run
+   * Class 3 = Hypoalgesia
+   * Class 4 = Neutral suggestion in hypoalgeisa run
+
+![](images/confusion_matrix_finalSVC.png)
+
+* With the final model, all the coefficients were reprojected to the MNI space using `masker.inverse_transform()`. Since there were four classes, six different class comparisons can be made and the highest coefficient for each comparison are presented in the plots below. The highest peaks represents the brain regions that either when activated or deactivated (red or blue) had the highest preictive value of the class(type of suggestion). For example, when looking at the first plot, the activation of the dlPFC apears to be the most predictive to predict the Neut_HYPO vs HYPO class.
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:152]: typo appears

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+![](coeff5.png)
+![](coef6.png)
+
+## Conclusion
+
+The main objective of this project was to complete a decoding model having the voxels of statistical brain maps as features and type of verbal suggestions as classes. Globally this project and it objectives have been reached. The delivarables were mostly acheived. Also, althought the code to train the decoding model has been made and is operational, the statistical maps that were used to train the model are prone to change in the sense that quality-control is still required in order to be confident about the validity of the results.
+
+## Acknowledgement
+
+Thanks to all the instructors, espicially Marie-Eve and Francois who took the time to sit down with me and precise some concepts and shed some light on many topics such as the proper workflow, bebugging techniques. proper use of sklearn, nilearn modules and so on!
```
Comment (me-pic) [inline content/en/project/decoding-pain-related-brain-activity/index.md:167]: debugging techniques,

**Review by me-pic — CHANGES_REQUESTED** [review]
Please check some typos

**Comment by dylansutterlin** [comment]
I pushed the requested changes!

File: content/en/project/decoding-pain-related-brain-activity/index.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+summary: "Painful experience involves a distributed pattern of brain activity. With hypnosis, it's possible to increase or decrease pain. This project aims to decode fMRI pain-evoked brain activity and identify pattern of activity that are associated with specific hypnotic conditions"
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "brain-activity.webp"
+---
+### Background
+#### personal background
+
+Link to Github page : <a href="https://github.com/dylansutterlin">
```
Comment (pbellec) [inline content/en/project/decoding-pain-related-brain-activity/index.md:31]: this should link to the github of the project, not a personal page.

**Comment by me-pic** [comment]
@pbellec All my comments have been addressed. I found some typo but I guess that codespell will fix them

**Comment by pbellec** [comment]
Thanks @me-pic merging...

**Pull request merged**